.github/workflows/label_checker.yml

```diff
@@ -7,16 +7,28 @@ on:
       - reopened
       - labeled
       - unlabeled
+  issues:
+    types:
+      - milestoned
+      - demilestoned
```

Comment:
I don't understand why this workflow needs to run when issues change? Or is this because PRs are a type of issue and these categories only apply to issues?

Reply:
For GitHub issues means Issues and Pull Requests (if you check the Rest API, part of operations on PR is done using Issues endpoints). 

There are no milestone-related events only for pull requests.

Reply:
If you remember the early GitHub time, you were able to create and issue and then attached code to it, and it would _become_ a PR. It was a mess becaues then you couldn't close the PR but keep the issue. Since then API have always mixed issues and PRs in a lot of places.

Reply:
I actually don't remember such a time @Carreau, you got in at least one generation ahead of me. 😃  ([update](https://caius.github.io/github_id): about 150K users before me.:smiley:)